lab-justin/lib/bodyParser.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = function(req){
+  return new Promise(function(resolve, reject){
+    req.body = '';
+    req.on('data', function(data){
+      req.body +=data.toString();
```

Comment:
Put spaces around operators to make them easier to read. Otherwise, this looks good!